crates/bifrost/src/providers/replicated_loglet/sequencer/mod.rs

```diff
@@ -158,6 +162,45 @@ impl<T: TransportConnect> Sequencer<T> {
         self.record_permits.available_permits()
     }
 
+    /// wait until all in-flight appends are drained. Note that this will cause the sequencer to
+    /// return AppendError::Sealed for new appends but it won't start the seal process itself. The
+    /// seal process must be started externally and _only_ after the drain is complete, the caller
```

Comment:
The comment says that the seal process should only be started after we completed the `drain` call. In the body of the method, there is a comment saying that we assume that seal has been executed already. I guess the latter is correct given the callers of this method.

Reply:
I probably should rephrase. The command says that seal must be started externally (before drain) but the drain bit (I should rephrase) (i.e known_global_tail.is_sealed()) must only be set after draining.

Reply:
Updated the phrasing a little.